apps/mobile/src/hooks/use-create-wallet.ts

```diff
@@ -37,6 +39,7 @@ export function useCreateWallet() {
           }),
         });
         router.navigate(AppRoutes.Home);
+        await registerNotifications();
```

Comment:
Do we really want to registerNotifications every time a wallet is created? Not sure what the right solution is but calling `registerNotifications` all over the place feels like a code smell to me - feels like `createWallet` is a leaky abstraction. Not necessarily asking you to change this now but can we think of a more sustainable approach here so that we don't keep adding side effects to `createWallet` in the future?